config/routes.rb

```diff
@@ -1,3 +1,4 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+	resources :centers
+  root 'centers#index'
```

Comment:
!!! indentions have been fixed but changes are not reflected on github for unknown reason.